papers/hendrik_heuer/heuer.rst

```diff
@@ -0,0 +1,223 @@
+:author: Hendrik Heuer
+:email: hendrik.heuer@aalto.fi
+:institution: Aalto University, Finland
+
+------------------------------------------------------------------------------
+Text comparison using word vector representations and dimensionality reduction
+------------------------------------------------------------------------------
+
+.. class:: abstract
+
+   This paper describes a technique to compare large text sources using word vector representations (word2vec) and dimensionality reduction (t-SNE) and how it can be implemented using Python. The technique provides a bird’s-eye view of text sources, e.g. text summaries and their source material, and enables users to explore text sources like a geographical map. Word vector representations capture many linguistic properties such as gender, tense, plurality and even semantic concepts like "capital city of". Using dimensionality reduction, a 2D map can be computed where semantically similar words are close to each other. The technique uses the word2vec model from the gensim Python library and t-SNE from scikit-learn.
+
+.. class:: keywords
+
+   Text Comparison, Topic Comparison, word2vec, t-SNE
+
+Introduction
+------------
+
+When summarizing a large text, only a subset of the available topics and stories can be taken into account. The decision which topics to cover is largely editorial. This paper introduces a tool that assists this editorial process using word vector representations and dimensionality reduction. It enables a user to visually identify agreement and disagreement between two text sources. 
+
+There is a variety of different ways to approach the problem of visualizing the topics present in a text. The simplest approach is to look at unique words and their occurrences and visualize the words in a list. Topics could also be visualized using word clouds, where the font size of a word is determined by the frequency of the word. Word clouds have a variety of shortcomings: They can only visualize small subsets, they focus on the most frequent words and they do not take synonyms and semantically similar words into account.
+
+This paper describes a human-computer interaction-inspired approach of comparing two text sources. The approach yields a bird’s-eye view of different text sources, including text summaries and their source material, and enables users to explore a text source like a geographical map.
+As similar words are close to each other, the user can visually identify clusters of topics that are present in the text. Conceptually, it can be understood as a "Fourier transformation for text".
```

Comment:
The metaphor was inspired by this animation:

https://commons.wikimedia.org/wiki/File:Fourier_transform_time_and_frequency_domains.gif

I decided to remove it for clarity.